action.yml

```diff
@@ -0,0 +1,27 @@
+name: install-dylan-tool
+description: Install a specific version of dylan-tool
+inputs:
+  version:
+    type: string
+    # Is there a way to specify the latest released version?
```

Comment:
The only thing I can think of is to create a (lightweight) git tag that points to the latest release and then to force-update it every time a new release is tagged.